BlazarUI/app/scripts/components/shared/PageHeader.jsx

```diff
@@ -1,6 +1,19 @@
 import React, {Component, PropTypes} from 'react';
+import $ from 'jquery';
```

Comment:
https://github.com/captivationsoftware/react-sticky is this useful here?

:+1: to not falling back to jQuery when the animation stuff isn't obvious w/ React.